include/SFML/Graphics/Texture.hpp

```diff
@@ -421,6 +421,18 @@ class SFML_GRAPHICS_API Texture : GlResource
     Texture& operator =(const Texture& right);
 
     ////////////////////////////////////////////////////////////
+    /// \brief Get the underlying OpenGL handle of the texture.
+    ///
+    /// You shouldn't need to use this function, unless you have
+    /// very specific stuff to implement that SFML doesn't support,
+    /// or implement a temporary workaround until a bug is fixed.
+    ///
+    /// \return OpenGL handle of the texture or 0 if not yet created
+    ///
+    ////////////////////////////////////////////////////////////
+    unsigned int getNativeHandle() const;
```

Comment:
Unlike with the shaders, on all systems (including OS X), the handle type for textures is an integer value (specified in OpenGL specification), so no casting needed here.